flink-python/pyflink/table/table_environment.py

```diff
@@ -1280,7 +1297,8 @@ def connect(self, connector_descriptor):
     @staticmethod
     def create(execution_environment=None, table_config=None, environment_settings=None):
         """
-        Creates a :class:`~pyflink.table.BatchTableEnvironment`.
+        Creates a :class:`~pyflink.table.BatchTableEnvironment` for a
```

Comment:
Revert the change as we can also create the TableEnvironment without the execution_environment.